src/main/java/com/yscope/logging/log4j1/EnhancedAppenderSkeleton.java

```diff
@@ -1,34 +1,43 @@
 package com.yscope.logging.log4j1;
 
 import org.apache.log4j.AppenderSkeleton;
+import org.apache.log4j.helpers.ISO8601DateFormat;
 import org.apache.log4j.helpers.LogLog;
 
+import java.util.Date;
```

Comment:
Java imports should go before 3rd party imports.

Reply:
Done. Will raise another pull request to fix import ordering for other files.